CDCgov/cdc-react
# Portal 1150: use localized web fonts

This PR makes sure the UI is using the correct fonts by linking to localized web fonts. This PR also fixes the issue where the base font declarations were being applied at a global level, causing any downstream modifications to be overwritten.

To test that the fonts are working as expected:

1. Run `yarn storybook` and look at the rendered page
2. The page should default to the button component. Right click on the "Default Button:" label (this should be a `h5` element) and Inspect to open the dev tools.
3. Ensure locally installed fonts are disabled by clicking the three dots, "More tools," and "Rendering."
4. In the "Rendering" panel, click the checkbox next to the "Disable local fonts" option. With the panel still open, refresh the window to apply the changes.
<img width="553" alt="Screenshot 2023-11-13 at 8 58 57 AM" src="https://github.com/CDCgov/cdc-react/assets/70541324/efc0dd2f-db7b-445c-b608-a6ee68c1cfcf">
<img width="551" alt="Screenshot 2023-11-13 at 8 59 15 AM" src="https://github.com/CDCgov/cdc-react/assets/70541324/ee9bca37-e722-4380-9746-9fa96ec6da63">
<img width="549" alt="Screenshot 2023-11-13 at 8 59 37 AM" src="https://github.com/CDCgov/cdc-react/assets/70541324/da2d72ec-f02c-40d4-a515-7d802c1797e9">

5. Navigate to the "Elements" tab in the dev tools panel, and then to the "Computed" sub tab. Scroll to the very bottom to the "Rendered Fonts" section. You should see "Open Sans"
<img width="385" alt="Screenshot 2023-11-14 at 11 52 46 AM" src="https://github.com/CDCgov/cdc-react/assets/70541324/bcd6a809-b579-4b7d-b3b1-d2857b76c279">


Doing this same process on the current live storybook instance will show the difference, that Helvetica is actually being rendered
<img width="419" alt="Screenshot 2023-11-14 at 11 53 08 AM" src="https://github.com/CDCgov/cdc-react/assets/70541324/5f0cc74e-9303-4a45-9a01-6ced8fca31e2">
